Vonk.Plugin.DocumentOperation.Test/DocumentOperationTests.cs

```diff
@@ -209,6 +209,7 @@ public async Task DocumentOperationInternalServerErrorOnMissingReference1()
 
             // Check response status
             testContext.Response.HttpResult.Should().Be(StatusCodes.Status500InternalServerError, "$document should return HTTP 500 - Internal Server error when a reference which is referenced by the composition can't be resolved");
+            testContext.Response.Outcome.Issue.Count(issue => issue.Code == OperationOutcome.IssueType.NotFound).Should().NotBe(0);
         }
```

Comment:
Try to always include a failMessage: easier to debug if any of the assertions fail.